### PR TITLE
Add angular-formly

### PR DESCRIPTION
**Pull request by kentcdodds**

### CODE REVIEW

File: _data/projects/angular-formly.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: angular-formly
+desc: JavaScript powered forms for AngularJS
+site: http://angular-formly.com
+tags:
+- angular
+- forms
+- JavaScript
+- angularjs
+- angular-formly
```
Comment (shiftkey) [inline _data/projects/angular-formly.yml:9]: probably don't need this one (the other terms should cover it) but up to you...

**Comment by shiftkey** [comment]
Just that one little bit of feedback. Everything else is :gem:

File: _data/projects/angular-formly.yml (hunk 1 above)
Comment (kentcdodds) [inline _data/projects/angular-formly.yml:9]: I don't mind if you don't

Comment (shiftkey) [inline _data/projects/angular-formly.yml:9]: :thumbsup:

**Comment by ritwik12** [comment]
@kentcdodds Following up with #1015. We noticed that your repo is archived and is read-only. Can you please send a PR for removing your project from the website if it is going to be archived in future. If you have plans to unarchive it then please tell us.

**Comment by kentcdodds** [comment]
In the last 3 years I've stopped maintaining the project and nobody stepped up to maintain it so it's archived and I don't have any plans to unarchive it.

**Comment by ritwik12** [comment]
@kentcdodds Thanks for letting us now. I am going to remove it from our list so that others will not get confused.

**Pull request merged**